stock_picking_operation_quick_change/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Stock Picking Operation Quick Change",
+    "summary": "Change location of all picking operations",
+    "version": "12.0.1.0.0",
+    "category": "Inventory",
```

Comment:
Change this to 'Warehouse' to get it in good category in odoo-community.org/shop

Reply:
done